app/assets/javascripts/autocomplete.coffee

```diff
@@ -0,0 +1,69 @@
+delay = do ->
+  timer = 0
+  (callback, ms) ->
+    clearTimeout timer
+    timer = setTimeout(callback, ms)
+    return
+
+update_textfield = (list_element) ->
+  # remove focused state
+  $(list_element).removeClass('suggestion-focused')
+  $('#autocomplete-textfield').val($(list_element).find('.res-name')[0].innerHTML)
+
+ready = ->
+  # TODO: test with IE 8
+  # onpropertychange -> oninput for IE 8
+  # https://msdn.microsoft.com/en-us/library/ms536956(v=vs.85).aspx
+  $('#autocomplete-textfield').on('focus input propertychange', ->
```

Comment:
GitHub doesn't support IE 8 https://help.github.com/articles/supported-browsers/, so we don't have to worry about testing for that.